nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/main/resources/docs/org.apache.nifi.atlas.reporting.ReportLineageToAtlas/additionalDetails.html

```diff
@@ -275,7 +276,7 @@ <h3 id="provenance-events">NiFi Provenance Event Analysis</h3>
             </ul>
         </p>
 
-        <p>'clusterName' in 'qualifiedName' attribute is resolved by mapping ip-address or hostname available at NiFi Provenance event 'transitUri' to a cluster name. See <a href="cluster-name">Cluster Name Resolution</a> for detail.</p>
+        <p>'namespace' in 'qualifiedName' attribute is resolved by mapping ip-address or hostname available at NiFi Provenance event 'transitUri' to a cluster name. See <a href="namespaces">Namespaces</a> for detail.</p>
```

Comment:
There are still some clusterName references in the documentation below.
Eg.: `remoteOutputPortGUID@clusterName`
Could you please fix those too?